tools/creality_printer_test_server.py

```
@@ -190,6 +200,132 @@ def _gen():
+    without a keyframe either, so the HLS/recording path looked broken against
+    the simulator even though real printers work.
+
+    Real Creality cameras emit a keyframe roughly every second, so a short clip
+    is pre-encoded with `keyint=fps` and its packets are looped. aiortc detects
+    an `av.Packet` (rather than a `VideoFrame`) and packetises it directly,
+    skipping its own encoder entirely.
+    """
+
+    kind = "video"
+
+    def __init__(self, width: int, height: int, fps: int, ffmpeg_bin: str = "ffmpeg",
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)